js/rendererjs/pluginapi.js

```diff
@@ -1,16 +1,18 @@
 // pluginapi.js: Sia-UI plugin API interface exposed to all plugins.
 // This is injected into every plugin's global namespace.
-import Siad from 'sia.js'
+import * as Siad from 'sia.js'
 import { remote } from 'electron'
 import React from 'react'
 import DisabledPlugin from './disabledplugin.js'
 const dialog = remote.dialog
 const mainWindow = remote.getCurrentWindow()
 const config = remote.getGlobal('config')
-Siad.configure(config.siad)
+const siadConfig = config.siad
 let disabled = false
 
-window.onload = () => {
+const sleep = (ms = 0) => new Promise((r) => setTimeout(r, ms))
+
+window.onload = async function() {
```

Comment:
why does the `onload` function need to marked `async`? Do you need to mark the function `async` if it uses `await` in the body?